src/index.js

```diff
@@ -131,9 +149,11 @@ export class WT {
   }
 
   getEventEnvironmentArgs() {
+    const pageUuid = this.pageUuid || this.getUUIDToken();
```

Comment:
let's not do the `|| this.getUUIDToken()` because the instance variable will always be set